src/utils/columnUtils.ts

```diff
@@ -15,7 +15,7 @@ export function getColumnMetrics<R>(metrics: Metrics<R>): ColumnMetrics<R> {
   let allocatedWidths = 0;
   let unassignedColumnsCount = 0;
   let lastFrozenColumnIndex = -1;
-  const columns: Array<Column<R> & { width: number | void }> = [];
+  const columns: Array<Column<R> & { width: number | undefined }> = [];
```

Comment:
`void` was failing for some reason

Reply:
I think `void` means no data type, `undefined` is a valid `nothing exists` type. They are still slightly different.